integration-tests/README.md

```diff
@@ -22,11 +24,28 @@ cargo run --release -- --dev --mining-key 5DkPYq8hFiCeGxFBkz6DAwnTrvKevAJfTYrzFt
 docker run --rm -it -p 8545:8545 gluwa/hardhat-dev
 ```
 
+2. Update creditcoin-js type definitions
+
+```bash
+    pushd creditcoin-js/
+
+    ./get-metadata.sh
+    yarn build:types
+    yarn format
+```
+
+**Important:**
+
+These are updated automatically in CI. If you are not sure that your
+branch is up-to-date or if you are modifying Creditcoin types then execute
+this step!
+
+**Warning:** don't forget to commit changed definitions to git!
+
 2. Install dependencies:
```

Comment:
There are two twos. It is somewhat confusing.

Reply:
I was hoping for the markdown rendering engine to render these sequentially without having to modify the rest of the items but it didn't :-(. So I modified the numbering.